test/e2e/autoscaling/custom_metrics_stackdriver_autoscaling.go

```diff
@@ -39,6 +40,8 @@ const (
 	stackdriverExporterDeployment = "stackdriver-exporter-deployment"
 	dummyDeploymentName           = "dummy-deployment"
 	stackdriverExporterPod        = "stackdriver-exporter-pod"
+	externalMetricValue           = 85
```

Comment:
nit: it has to be converted to int64 in every test, we could probably just declare it as this type

Reply:
Done